code/src/styling/index.css

```diff
@@ -0,0 +1,207 @@
+
```

Comment:
The CSS looks well organized and easy to read, nice use of grid!
Nothing really to comment on, I have heard that it's a good thing to use `rem` instead of `px` when sizing fonts - maybe try that out in some other project? I know atleast I need to practise that.